src/components/Tutorials/NewTutorial/index.jsx

```diff
@@ -252,7 +253,14 @@ const NewTutorial = ({ viewModal, onSidebarClick, viewCallback, active }) => {
                 loading={loading}
                 onClick={e => onSubmit(e)}
                 data-testid="newTutorialSubmit"
-                style={{ backgroundColor: "#03AAFA", borderRadius: "30px" }}
+                sx={{
+                  bgcolor: '#03AAFA',
+                  borderRadius: '30px',
+                  color: common.white,
+                  '&:hover': {
+                    bgcolor: '#03AAFA', 
```

Comment:
i have added this hover because the default hover of purple was applied when i used sx